ta/miss/ta_miss.c

```diff
@@ -0,0 +1 @@
+../sims/ta_sims.c
```

Comment:
No symbolic link please, they can cause all sorts of troubles. Use #include "../sims/ta_sims.c" instead.